### DISCUSSION TITLE
Is it possible to have the darkening window appear above/below waybar?

### DISCUSSION THREAD

**Discussion by auctumnus** [ErikReider/SwayNotificationCenter#193] [Q&A]
I would open an issue, but I'm not sure if this is possible under Wayland. At the moment, I'm running Hyprland with Waybar, and when I open the control center, the darkening window (and the control center as a whole) are under Waybar:

![image](https://user-images.githubusercontent.com/37276280/210180259-7d383dba-905f-4e18-991b-701b42130a80.png)

This obviously doesn't look very good. I could remove the darkening window, but I think it'd be better if it were possible to have it render such that it also darkens Waybar, or appears on a layer below it.

This could also just be a problem with my Waybar configuration.

**Comment by ErikReider** [comment]
Yeah. I can't really do anything about this. This wasn't an issue in the past until #136 which is due to in which order each Wayland WM opens new layer windows...

Displaying it above Waybar would also look strange :/
